tasks/bundle-js.js

```diff
@@ -100,27 +100,27 @@ async function bundleJS(/** @type {JSEntry} */entry, {production}) {
                 tsconfig: 'src/tsconfig.json',
                 tsconfigOverride: {
                     compilerOptions: {
-                        removeComments: production ? true : false,
-                        sourceMap: production ? false : true,
+                        removeComments: debug ? false : true,
+                        sourceMap: debug ? true : false,
                     },
                 },
-                clean: production ? true : false,
-                cacheRoot: production ? null : `${fs.realpathSync(os.tmpdir())}/darkreader_typescript_cache`,
+                clean: debug ? false: true,
+                cacheRoot: debug ? `${fs.realpathSync(os.tmpdir())}/darkreader_typescript_cache` : null,
             }),
             rollupPluginReplace({
-                '__DEBUG__': production ? 'false' : 'true',
-                '__PORT__': production ? '-1' : String(PORT),
+                '__DEBUG__': watch ? 'true' : 'false',
+                '__PORT__': watch ? String(PORT) : '-1',
```

Comment:
Add a 3rd `__WATCH__` parameter here and use if to enable or disable the WebSocket connection.